aspnetcore/migration/fx-to-core/start.md

```diff
@@ -154,7 +154,6 @@ The [System.Web adapters](~/migration/fx-to-core/inc/systemweb-adapters.md) can
 1. Remove reference to `System.Web` in the project file
 2. Add the `Microsoft.AspNetCore.SystemWebAdapters` package
 3. Enable multi-targeting and add a .NET 8 target or later, or convert the project to .NET Standard 2.0.
-4. Ensure the target framework supports .NET Core. Multi-targeting can be used if .NET Standard 2.0 is not sufficient
```

Comment:
Bullet 4 was deleted that's probably OK but I don't know

Reply:
Yes, I removed this bullet because I believe it's redundant.